### ISSUE TITLE
Feature request: Add to-do from Kanban

### ISSUE THREAD

**Issue by nome** [joplin/plugin-kanban#20]
First of all, thank you! This plugin does almost perfectly what I expect from a task management app, it's the main reason I started using Joplin.

There's just one little issue: It's possible to create notes directly from the Kanban board (cf. #3); however, I'd like the tasks created by the `+` buttons to be to-do items instead of notes (yes I could track completion with a tag, but it's more convenient to have a checkbox for that, plus Joplin renders completed to-dos in a different style, which makes it very easy to get a per-subnotebook overview of task completion). EDIT: not to forget, to-do items can have alarms, while notes cannot.

This could be configuration option, e.g. `task_type: todo`, defaulting to `task_type: note`.

**Comment by mablin7** [comment]
Try adding `completed: false` to the filters. After the latest update, this could work the way you want it

**Comment by amandamcg** [comment]
I tried adding (code below in case I misunderstood), but that just hid the notes in my done column. New notes in any column still seem to be "notes" not "to-dos". Version 1.0.7

```
```kanban
filters:
	completed: false 
```

**Comment by nome** [comment]
I've updated the kanban plugin (to version 1.0.7), and added a `completed: false` filter. I'm seeing the same effects @amandamcg described, so I'm afraid this doesn't help me.

Full config:
```kanban
columns:
  - name: Backlog
    backlog: true
  - name: Next
    tag: next
  - name: Work in progress
    tag: wip
  - name: Finished
    completed: true
filters:
  completed: false
```

**Comment by mablin7** [comment]
I see, I'll check it out

**Comment by mbissinger** [comment]
If I understand correctly then having another button in the header for creating to-do items might be an option? For instance using the same symbols as already shipped by Joplin:

![image](https://user-images.githubusercontent.com/26654668/156318236-c9ee40f3-edaa-41b0-9e9a-8f6fae3b782b.png)

One might even think about a context menu of an item on the board (right-click) with the ability to complete a to-do. This would be extremely convenient, but I'm note sure if this is possible. Another option would be to have checkbox for to-do items like

![image](https://user-images.githubusercontent.com/26654668/156320083-b0a532fa-807b-4956-819a-e8671118ab1c.png)

**Comment by amandamcg** [comment]
Yes, 2 buttons would work. Or a setting to choose which to appear, if that takes too much space. 

Re: completing a todo, I use the done column for that, but could see others wanting to have that as a separate option.
```
- name: ✅ Done
    completed: true
```

**Comment by nome** [comment]
I agree, two buttons would be a perfectly valid solution. I also use the done/finished column to complete to-do items, so as far as I'm concerned the "completing via kanban" feature is already covered.

**Comment by dreamflasher** [comment]
I came here to request the following:
* Add a To-Do checkbox in front of the items that are todos, to allow for quickly marking them completed (I don't have an additional Done column, because 1) I am not interested in seeing the done items, 2) I used the board as categories, so I want to keep the category tags)
* Quickly add new To-Dos by having a text-entry field below the last item, that creates a new note of the correct type (todo if the lane has a completed filter, else a normal note) when hitting enter
